web_external/templates/widgets/studyResultsStudyDetailPage.jade

```diff
@@ -0,0 +1,15 @@
+.isic-study-results-study-details-container
+  .isic-study-results-header Study Details
+  if hasStudy
+    - var featureset = model.get('featureset')
+    table.table.table-condensed.isic-study-results-study-details-table
+      tr
+        td Unique ID
```

Comment:
I realized this should have the `.isic-study-results-study-details-table-label` class.